teams/wg-pgo.toml

```diff
@@ -5,9 +5,9 @@ kind = "working-group"
 [people]
 leads = ["michaelwoerister"]
 members = ["michaelwoerister"]
-alumni = []
+alumni = ["michaelwoerister"]
 
 [website]
 name = "Profile-guided optimization working group"
-description = "Implementing profile-guided optimization for rustc"
```

Comment:
We might put this team into the archive.

Thanks for all your work on Rust, btw! :heart:

Reply:
Done